### ISSUE TITLE
Best Buy having Store Pickup only

### ISSUE THREAD

**Issue by pilotw7** [natewong1313/bird-bot#19]
How will this bot work when currently Best Buy is only doing store pickup for the Nintendo Switch? I don't want it to order a switch only for it to be at a location that is far away..

**Comment by jmdelvalle22** [comment]
All you gotta do is simple... stop being a fucking dumbass and give everyone a chance to buy the damn thing the NORMAL way... by PAYING what it actually costs in store instead of trying to sell it for $700. Get your lazy ass to best buy and pick it up your damn self

**Comment by Victor-Villacis** [comment]
Interesting

**Comment by nicolas-guerra** [comment]
Yeah I have no idea. Is there an option to only have best buys a certain distance?
